scss/modules/_grid.scss

```diff
@@ -9,38 +9,73 @@ $grid-col-padding: 15px;
   .container-fluid { width: 100%; }
 
   .container {
+    margin: 0 auto;
```

Comment:
Don't see a reason to set vertical margin here. If it's just a shorthand to replace what we had I think it has unnecessary side-effects.